programs/migrations/0096_auto_20241107_1221.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 4.2.15 on 2024-11-07 17:21
+
+from django.db import migrations
+from integrations.services.google_translate.integration import Translate
+
+
+def add_apply_button_description(apps, schema_editor):
+    Program = apps.get_model("programs", "Program")
+    Translation = apps.get_model("translations", "Translation")
+
+    for program in Program.objects.all():
+        translation = Translation.objects.add_translation(
+            f"programs.{program.name_abbreviated}-{program.id}_apply_button_description", "Apply Online"
+        )
+
+        auto_translations = Translate().bulk_translate(["__all__"], ["Apply Online"])["Apply Online"]
```

Comment:
Because we are defaulting to "Apply Online" on the frontend, lets make all of the current translations empty strings. As it is right now, you will need a Google Translate API key to run migrations which is not ideal for developers coming on to the project.